media-libs/shaderc/files/shaderc-2017.2-disable-git-versioning.patch

```diff
@@ -0,0 +1,26 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
```

Comment:
Looks non-upstreamable. Could you try providing something with `option(...)` switch that upstream could accept? ;-)

Reply:
I really don't mind having a non-upstreamble patch in this particular case.